lib/include/openamp/virtqueue.h

```diff
@@ -57,6 +57,15 @@ struct vq_desc_extra {
 	uint16_t ndescs;
 };
 
+#if defined(HVL_VIRTIO)
```

Comment:
please add comment to explain the need/use

Reply:
Will move to new PR